workflow/tests/base_results/results.csv

```diff
@@ -97,7 +97,7 @@ base-dhw-solar-fraction.xml,53.024,53.024,29.921,29.921,23.104,0.0,0.0,0.0,0.0,0
-base-dhw-tank-detailed-setpoints.xml,58.727,58.727,35.902,35.902,22.824,0.0,0.0,0.0,0.0,0.0,0.0,0.377,0.0,0.0,4.273,0.823,9.153,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,22.824,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,21.375,0.0,13.981,9.207,0.623,0.0,0.0,0.0,0.0,2477.2,3308.7,23.031,17.893,0.0,3.542,3.634,0.511,7.499,0.628,10.505,-12.551,0.0,0.0,0.0,8.272,-0.062,4.804,0.0,0.728,0.0,4.939,-8.91,-2.499,0.0,-0.043,-0.455,-0.051,2.706,-0.024,-1.917,11.732,0.0,0.0,0.0,-6.315,-0.059,-1.166,-3.065,-0.165,0.0,3.097,7.877,2.01,1354.8,997.6,11442.2,2625.6,36000.0,24000.0,0.0,6.8,91.76,31792.0,8583.0,7508.0,0.0,575.0,6409.0,0.0,0.0,1949.0,2171.0,4597.0,18786.0,5328.0,7037.0,0.0,207.0,265.0,0.0,0.0,0.0,2010.0,619.0,3320.0,0.0,0.0,0.0,0.0
+base-dhw-tank-detailed-setpoints.xml,58.727,58.727,35.902,35.902,22.824,0.0,0.0,0.0,0.0,0.0,0.0,0.377,0.0,0.0,4.273,0.823,9.153,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,22.824,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,21.375,0.0,13.981,9.207,0.623,0.0,0.0,0.0,0.0,2477.3,3308.7,23.031,17.893,0.0,3.542,3.634,0.511,7.499,0.628,10.505,-12.551,0.0,0.0,0.0,8.272,-0.062,4.804,0.0,0.728,0.0,4.939,-8.91,-2.499,0.0,-0.043,-0.455,-0.051,2.706,-0.024,-1.917,11.732,0.0,0.0,0.0,-6.315,-0.059,-1.166,-3.065,-0.165,0.0,3.097,7.877,2.01,1354.8,997.6,11442.2,2625.6,36000.0,24000.0,0.0,6.8,91.76,31792.0,8583.0,7508.0,0.0,575.0,6409.0,0.0,0.0,1949.0,2171.0,4597.0,18786.0,5328.0,7037.0,0.0,207.0,265.0,0.0,0.0,0.0,2010.0,619.0,3320.0,0.0,0.0,0.0,0.0
```

Comment:
Are there just some rounding differences in here somewhere? And for the sample files below?